hebiros_gazebo_plugin/include/hebiros_gazebo_pid.h

```diff
@@ -0,0 +1,25 @@
+#include "hebiros/PidGainsMsg.h"
+#include <cstdint>
+
+namespace hebiros {
+namespace sim {
+
+// A simple PID controller class
+class PidController {
+
+public:
+  PidController(double ff_scale)
```

Comment:
(minor thing) I've tried to force myself to make every constructor with a single argument marked `explicit` to prevent any implicit constructions

Reply:
I'm not adamant about this changing, but it's up to you